include/client.h

```diff
@@ -591,19 +589,49 @@ class Client
         Redis* _redis;
 
         /*!
-        *   \brief Execute a Command
-        *   \param cmd The Command to execute
+        *   \brief Execute a non-keyed Command that addresses
+        *          the given db node(s)
```

Comment:
Small change, but should this be "Execute a non-keyed Command that address a given node" since the NonKeyedCommand class only holds one address and port?  This assumes we are going to accept multiple address and ports at the client level and not the Command level.